windtunnel_dataset/download_simulations.py

```diff
@@ -0,0 +1,135 @@
+"""Download and postprocess WindTunnel simulations."""
+
+import json
+import os
+
+import inductiva
+import windtunnel
+from absl import app, flags, logging
+
+FLAGS = flags.FLAGS
+flags.DEFINE_boolean("debug", False, "Enable debug mode")
+flags.DEFINE_string("submissions_file", "submissions.jsonl",
+                    "Path to the submissions file.")
+flags.DEFINE_string("data_folder", "data", "Path to the data folder.")
+flags.DEFINE_string("input_mesh_file", "input_mesh.obj",
+                    "Name of the input mesh file.")
+flags.DEFINE_string("openfoam_mesh_file", "openfoam_mesh.obj",
+                    "Name of the OpenFOAM mesh file.")
+flags.DEFINE_string(
+    "pressure_field_mesh_file",
+    "pressure_field_mesh.vtk",
+    "Name of the pressure field mesh file.",
+)
+flags.DEFINE_string(
+    "streamlines_mesh_file",
+    "streamlines_mesh.ply",
+    "Name of the streamlines mesh file.",
+)
+flags.DEFINE_string(
+    "simulation_metadata_file",
+    "simulation_metadata.json",
+    "Path to the simulation metadata file.",
+)
+flags.DEFINE_string(
+    "failed_simulations_file",
+    "failed_simulations.json",
+    "Path to the failed simulations file.",
+)
+
+
+def download_task(task_id):
+    task = inductiva.tasks.Task(task_id)
+    task.wait()
+    if task.is_failed():
+        raise Exception("Task failed")
+    output_dir = task.download_outputs()
+    return output_dir
+
+
+def postprocess_tasks():
+    with open(FLAGS.submissions_file, "r", encoding="utf-8") as file:
+        for line in file:
+            task_metadata = json.loads(line.strip())
+            task_id = task_metadata.get("task_id")
+            print(f"Processing task: {task_id}")
+            object_number = (
+                task_metadata.get("object_file").split("_")[-1].split(".")[0])
+            if int(object_number) < 700:
```

Comment:
I removed the train/validation/test splits from the dataset generation because I believe it was a very specific thing (even if it accepted percentages)